src/components/TodoContainer.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TodosList from './TodosList';
+import Header from './Header';
+import InputTodo from './InputTodo';
+
+export default class ToDoContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [],
+    };
+  }
+
+  // componentDidMount() {
+  //   fetch("https://jsonplaceholder.typicode.com/todos?_limit=10")
+  //     .then(response => response.json())
+  //     .then(data => this.setState({ todos: data }));
+  // }
```

Comment:
Please kindly comment on the code from the component file which is a bad practice to make your code look clean.